src/lm.jl

```diff
@@ -19,6 +19,9 @@ function updatemu!{V<:FPVector}(r::LmResp{V}, linPr::V)
 end
 updatemu!{V<:FPVector}(r::LmResp{V}, linPr) = updatemu!(r, convert(V,vec(linPr)))
 
+"""
+Measure of the model fit, weighted residual sum of squares for lm's
```

Comment:
This definition is too general for the first part. Anyway, I think you'd better leave this out as you're documenting the specific method for `LmResp`, which is identical to the only for `LinearModel`. I've opened a PR with detailed docs for the function in StatsBase (I'll try to get this merged soon): https://github.com/JuliaStats/StatsBase.jl/pull/146/